docs/cockpit.rst

```diff
@@ -23,29 +23,21 @@ the host you are running the ADB on.
 
 1. Start the ADB
 
-2. Log into the ADB
+2. Start cockpit
 
-   ``$ vagrant ssh``
+   ``$ vagrant service-manager restart cockpit``
```

Comment:
My test just now showed it working.  Cockpit was not started or enabled before I ran this command.

I agree that "restart" is a bad verb here, but we don't have "start" yet.